src/components/data-management/CellLevelUploadModal.jsx

```diff
@@ -15,45 +14,45 @@ import {
 } from 'antd';
 import { CheckCircleTwoTone, DeleteOutlined } from '@ant-design/icons';
 import Dropzone from 'react-dropzone';
-
+import { useSelector } from 'react-redux';
 import handleError from 'utils/http/handleError';
 import endUserMessages from 'utils/endUserMessages';
 import readFileToString from 'utils/upload/readFileToString';
+import UploadDetailsModal from 'components/data-management/UploadDetailsModal';
+import downloadCellLevelMeta from 'utils/data-management/downloadCellLevelMeta';
 
 const { Text, Title, Paragraph } = Typography;
 
 const CellLevelUploadModal = (props) => {
-  const { onUpload, onCancel, uploading } = props;
-
-  const [canUpload, setCanUpload] = useState(false);
-  const [filesList, setFilesList] = useState([]);
+  const {
+    onUpload, onCancel, uploading, cellLevelMetadata,
+  } = props;
 
-  useEffect(() => {
-    setCanUpload(filesList.length === 1);
-  }, [filesList]);
+  const [file, setFile] = useState(false);
```

Comment:
Do you ever use the actual file that is stored under `file` if its state is not `false`? I see that `file` is used to check if we should disable the upload modal (here: `disabled={!file}`). If we are never using the actual value of `file` in this state variable, change the lines that use `setFile` to set it to `true` and rename `[file, setFile]` to reflect that this is a boolean. If the actual value of the state variable `file` is used, then consider changing the default state to `null` or `{}` or `undefined`, because `file` is not a boolean.

Reply:
Yes it is used. Before uploading a file it is not in redux so everything in this file is used.

Reply:
I see. If a file is uploaded, but then it is removed, what will the value of `file` be?

Reply:
false

Reply:
ok, then using `false` should be fine